t/fixtures/01-jobs.pl

```diff
@@ -136,6 +136,27 @@
         settings   => [{key => 'DVD', value => '1'}, {key => 'DESKTOP', value => 'kde'}, {key => 'ISO_MAXSIZE', value => '4700372992'}, {key => 'ISO', value => 'openSUSE-Factory-DVD-x86_64-Build0048-Media.iso'}, {key => 'QEMUCPU', value => 'qemu64'}]
     },
     Jobs => {
```

Comment:
better don't create a new fixture and pollute the other tests, better create the job in the test where you need it.